package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "cql-exec-fhir-mongo",
+  "version": "1.0.0",
+  "description": "Provides a FHIR data provider for use with CQL",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint \"**/*.js\"",
+    "lint:fix": "eslint \"**/*.js\" --fix",
+    "prettier": "prettier --check \"**/*.js\"",
+    "prettier:fix": "prettier --write \"**/*.js\"",
+    "check": "npm run test && npm run lint && npm run prettier",
+    "test": "jest --silent --runInBand",
+    "test:coverage": "jest --collectCoverage --silent --runInBand",
+    "test:watch": "jest --watchAll --runInBand",
+    "test:watch:coverage": "jest --watchAll --runInBand --collectCoverage"
```

```suggestion
    "test": "jest",
    "test:coverage": "jest --collectCoverage",
    "test:watch": "jest --watchAll",
    "test:watch:coverage": "jest --watchAll --collectCoverage"
```

we don't need these; they were deqm-test-server specific flags